### DISCUSSION TITLE
Short video on building the gpt-home!

### DISCUSSION THREAD

**Discussion by ashepp** [judahpaul16/gpt-home#31] [Show and tell]
Hi JudahPaul16, Just wanted to share a short view on the build of your fantastic project. Very much appreciate all the patience, assistance and troubleshooting along the way. Hope it results in broader exposure for you and the project. Keep up the great work!
https://www.linkedin.com/posts/sheppardadam_hi-folks-i-wanted-to-share-a-fun-one-day-activity-7196612104043261954-qSFg?utm_source=share&utm_medium=member_desktop

**Comment by judahpaul16** [comment]
Thanks and great job! I really appreciate your interest in the project.

**Comment by ashepp** [comment]
You're welcome, if I can ever be helpful with your ongoing career pursuits
feel free to reach out on LinkedIn. Great job again!
